model/session_test.go

```diff
@@ -41,26 +39,6 @@ func TestSessionDeepCopy(t *testing.T) {
 	assert.Equal(t, 0, len(copySession.TeamMembers))
 }
 
-func TestSessionJson(t *testing.T) {
-	session := Session{}
-	session.PreSave()
-	json := session.ToJson()
-	rsession := SessionFromJson(strings.NewReader(json))
-
-	require.Equal(t, rsession.Id, session.Id, "Ids do not match")
-
-	session.Sanitize()
-
-	require.False(t, session.IsExpired(), "Shouldn't expire")
-
-	session.ExpiresAt = GetMillis()
-	time.Sleep(10 * time.Millisecond)
-
-	require.True(t, session.IsExpired(), "Should expire")
-
-	session.SetExpireInDays(10)
-}
```

Comment:
How is this test related? Apart from removing the `SetExpireInDays()` call I am guessing this can be left as-is.

Reply:
Yeah, I guess so. Will change.